components/FlowCounter/index.js

```diff
@@ -0,0 +1,18 @@
+import useStore from "../useStore";
+import { StyledRoundCounter } from "../../styles/StyledRoundCounter";
+
+export default function FlowCounter() {
+  const flowCounter = useStore((state) => state.flowCounter);
+
+  return (
+    <>
+      {flowCounter < 1 ? (
+        <StyledRoundCounter>Round 1/3</StyledRoundCounter>
+      ) : flowCounter < 2 ? (
+        <StyledRoundCounter>Round 2/3</StyledRoundCounter>
+      ) : (
+        <StyledRoundCounter>Round 3/3</StyledRoundCounter>
+      )}
+    </>
```

Comment:
I would do something like this instead (maybe find a better way to check the condition):

```suggestion
    const currentFlowCounter = flowCounter < 1 ? "1" : flowCounter < 2 ? "2" : "3";

    return <StyledRoundCounter>Round {currentFlowCounter}/3</StyledRoundCounter>;
```